dipy/viz/actor.py

```diff
@@ -0,0 +1,661 @@
+        these are considered as the values to be used by the colormap per
+        streamline.
+        If an array (X, Y, Z) or (X, Y, Z, 3) is given then the values for the
+        colormap are interpolated automatically using trilinear interpolation.
+
+    opacity : float, optional
+        Default is 1.
+
+    linewidth : float, optional
+        Line thickness. Default is 1.
+    spline_subdiv : int, optional
+        Number of splines subdivision to smooth streamtubes. Default is None
+        which means no subdivision.
+    lod : bool
+        Use vtkLODActor(level of detail) rather than vtkActor. Default is True.
+        Level of detail actors do not render the full geometry when the
+        frame rate is low.
+    lod_points : int
+        Number of points to be used when LOD is in effect. Default is 10000.
+    lod_points_size : int
+        Size of points when lod is in effect. Default is 3.
+    lookup_colormap : bool, optional
+        Add a default lookup table to the colormap. Default is None which calls
+        :func:`dipy.viz.actor.colormap_lookup_table`.
+
+    Returns
+    ----------
+    v : vtkActor or vtkLODActor object
+        Line.
+
+    Examples
+    ----------
+    >>> from dipy.viz import actor, window
+    >>> ren = window.Renderer()
+    >>> lines = [np.random.rand(10, 3), np.random.rand(20, 3)]
+    >>> colors = np.random.rand(2, 3)
+    >>> c = actor.line(lines, colors)
+    >>> actor.add(ren, c)
+    >>> #window.show(ren)
+    """
+    # Poly data with lines and colors
+    poly_data, is_colormap = lines_to_vtk_polydata(lines, colors)
+    next_input = poly_data
+
+    # use spline interpolation
+    if (spline_subdiv is not None) and (spline_subdiv > 0):
+        spline_filter = set_input(vtk.vtkSplineFilter(), next_input)
+        spline_filter.SetSubdivideToSpecified()
+        spline_filter.SetNumberOfSubdivisions(spline_subdiv)
+        spline_filter.Update()
+        next_input = spline_filter.GetOutputPort()
+
+    poly_mapper = set_input(vtk.vtkPolyDataMapper(), next_input)
+    poly_mapper.ScalarVisibilityOn()
+    poly_mapper.SetScalarModeToUsePointFieldData()
+    poly_mapper.SelectColorArray("Colors")
+    poly_mapper.Update()
+
+    # Color Scale with a lookup table
+    if is_colormap:
+
+        if lookup_colormap is None:
+            lookup_colormap = colormap_lookup_table()
+
+        poly_mapper.SetLookupTable(lookup_colormap)
+        poly_mapper.UseLookupTableScalarRangeOn()
+        poly_mapper.Update()
+
+    # Set Actor
+    if lod:
+        actor = vtk.vtkLODActor()
+        actor.SetNumberOfCloudPoints(lod_points)
+        actor.GetProperty().SetPointSize(lod_points_size)
+    else:
+        actor = vtk.vtkActor()
+
+    # actor = vtk.vtkActor()
+    actor.SetMapper(poly_mapper)
+    actor.GetProperty().SetLineWidth(linewidth)
+    actor.GetProperty().SetOpacity(opacity)
+
+    return actor
+
+
+def lines_to_vtk_polydata(lines, colors=None):
```

Comment:
This is not an actor, we should move it in another file, probably `viz/utils.py`.